views/pages/fullEpisodes.ejs

```diff
@@ -0,0 +1,17 @@
+
+<ul>
+    <li><a href="/">Home</a></li>
+    <li><a href="/fullEpisodes">Full Episodes</a></li>
+    <li><a href="/Contact">Contact Us</a></li>
+</ul>
+
+
+<% for (let episode of episodeArray) { %>
+    <div class="card">
+        <a href="/singleEpisode/<%= episode.id%>">
+            <h3>Title: <%= episode.name %> <%= makeEpisodeCode(episode)%></h3>
+        </a>
+        <img src="<%= episode.image.medium %>" alt="Episode Cover Image"/>
```

Comment:
some episodes of some tv shows don't have object properties for episode.image so will cause your code to crash on this line (e.g. try the simpsons data).  That's a problem for future though!